gem/management/commands/remove_empty_nav_tags.py

```diff
@@ -11,9 +12,13 @@ def handle(self, **options):
         for article in ArticlePage.objects.all():
             for nav_tag in article.nav_tags.all():
                 if nav_tag.tag is None:
-                    nav_tag.delete()
-
-            if article.live:
-                article.save_revision().publish()
-            else:
-                article.save_revision()
+                    try:
+                        nav_tag.delete()
+                        if article.live:
+                            article.save_revision().publish()
+                        else:
+                            article.save_revision()
+                        print("nothing raised")
+                    except IntegrityError:
+                            return ("IntegrityError: Only articles with sites "
+                                    "can be saved")
```

Comment:
Please add something like the article id to this message. that way if we want to go and investigate further we know which articles are affected by this